.gitignore

```diff
@@ -24,3 +24,8 @@ Cargo.lock
 .DS_Store
 .python-version
 *.cpython-*.so
+.mypy_cache/
+.nox
+.pytest_cache
+.venv
+uv.lock
```

Comment:
Not sure about this, if at some point we use UV lockfiles, we might want to add them to the repo for reproduceability. I would delay de decision until we do it. If you need this locally you can always add it to `.git/info/excludes`.